maintenance/pin.sh

```diff
@@ -37,7 +37,6 @@ PY_38_SCRIPTS=(
 PY_39_SCRIPTS=(
     configloader
     scriptworker_client
```

Comment:
since treescript is the last script on 39, should we remove 39?

Reply:
That's coming next in #1049. :)